lib/createWorkers.js

```diff
@@ -83,7 +144,7 @@ function createWorkers(handlers, config, options = {}) {
     );
     yield workerChannels.map(channel => channel.close());
     if (workerConnection) yield workerConnection.close(forceExit);
-    if (forceExit) setTimeout(process.exit, configuration.processExitTimeout);
+    if (forceExit) setTimeout(_exit, configuration.processExitTimeout);
```

Comment:
Remove the `setTimeout` here if the comment on `_exit` is valid.

Reply:
yes